Ines/index.js

```diff
@@ -0,0 +1,28 @@
+let dogIsbetter = false;
+let catIsbetter = true;
+
+console.log("1.a is", dogIsbetter && catIsbetter);
+
+console.log("1.b is", dogIsbetter || catIsbetter);
+
```

Comment:
You missed 1c exercise! The solution is:
console.log( !(dogIsBetter && catIsBetter) );